standalone-metastore/metastore-server/src/main/sql/derby/hive-schema-4.0.0.derby.sql

```diff
@@ -629,7 +629,8 @@ CREATE TABLE COMPACTION_QUEUE (
   CQ_INITIATOR_ID varchar(128),
   CQ_INITIATOR_VERSION varchar(128),
   CQ_WORKER_VERSION varchar(128),
-  CQ_CLEANER_START bigint
+  CQ_CLEANER_START bigint,
+  CQ_RETRY_RETENTION integer NOT NULL DEFAULT 0
```

Comment:
I'm a little iffy about declaring this as an integer vs. a bigint, since we store milliseconds and this value could be 2^10 * hive.compactor.cleaner.retry.retentionTime which has no upper limit

Reply:
Good point, changed to bigint.